docs/roadmap/cosmos-hub-release-log.md

```diff
@@ -1,16 +1,22 @@
-# The Cosmos Hub Roadmap
+# Cosmos Hub Release Log
```

Comment:
@jtremback was concerned about breaking links to this document, was this change agreed upon?